zebra-chain/src/transaction/unmined.rs

```diff
@@ -116,8 +121,8 @@ impl UnminedTxId {
     #[allow(dead_code)]
     pub fn auth_digest(&self) -> Option<AuthDigest> {
         match self {
-            Narrow(_effect_id) => None,
-            Wide(wtx_id) => Some(wtx_id.auth_digest),
+            Legacy(_effect_id) => None,
```

Comment:
Minor: Same here? Maybe it could even be omitted:

```suggestion
            Legacy(_) => None,
```